borg/keymanager.py

```diff
@@ -77,16 +78,27 @@ def store_keyblob(self, args):
         elif self.keyblob_storage == KEYBLOB_REPO:
             self.repository.save_key(self.keyblob.encode('utf-8'))
 
+    def get_keyfile_data(self):
+        data = '%s %s\n' % (KeyfileKey.FILE_ID, bin_to_hex(self.repository.id))
+        data += self.keyblob
+        if not self.keyblob.endswith('\n'):
+            data += '\n'
+        return data
+
     def store_keyfile(self, target):
         with open(target, 'w') as fd:
-            fd.write('%s %s\n' % (KeyfileKey.FILE_ID, bin_to_hex(self.repository.id)))
-            fd.write(self.keyblob)
-            if not self.keyblob.endswith('\n'):
-                fd.write('\n')
+            fd.write(self.get_keyfile_data())
 
     def export(self, path):
         self.store_keyfile(path)
 
+    def export_qr(self, path):
+        with open(path, 'wb') as fd:
+            key_data = self.get_keyfile_data()
+            html = pkgutil.get_data('borg', 'paperkey.html')
```

Comment:
This works through the standard importlib; since `borg` was already imported... should be hard to break.